src/sentry/api/serializers/models/event.py

```diff
@@ -376,3 +376,36 @@ def serialize(self, obj, attrs, user):
             # Needed to generate minidump links in UI
             "crashFile": attrs["crash_file"],
         }
+
+
+class ExternalEventSerializer(EventSerializer):
+    """
+    Event serializer for the minimum event data needed to send to an external service. This
+    can be used for Integrations and the Integration Platform. If the full event is needed,
+    use `event.as_dict()` instead.
+    """
+
+    def serialize(self, obj, attrs, user):
+        tags = [{"key": key.split("sentry:", 1)[-1], "value": value} for key, value in obj.tags]
```

Comment:
Do you know why we strip the `sentry:` prefix from the `release` and `user` tags in some of the endpoints but not all of them? I think it would be good to avoid doing this if possible.

Reply:
I'm not entirely sure tbh